doc.txt

```diff
@@ -1,3 +1,20 @@
+/**
+ * @defgroup configurations Compile time configurations
+ * @brief    List of compile-time configurations
+ *
+ *           This group contains all compile-time configurations macros that are
+ *           intended to be modified by the user.
+ *
+ *           These configuration macros are defined within modules like this:
+ *           ```
```

Comment:
IIRC this fencing does not work for (all versions) of Doxygen. The right way (including syntax highlighting) would be

```
~~~~~~~~ {.c}
blafoo
~~~~~~~~
```